### ISSUE TITLE
Update JCE note in reference for recent JDK8 versions

### ISSUE THREAD

**Issue by bclozel** [spring-cloud/spring-cloud-config#851]
In recent JDK8 versions, it is not required to download and install JCE policies.

Editing `$JAVA_HOME/jre/lib/security/java.security` and making sure to uncomment `crypto.policy=unlimited` is enough.

Could you update the reference documentation to reflect that?

**Comment by ryanjbaxter** [comment]
@bclozel when you say "recent" do you mean only after a certain release or is it all JDK8 releases?

**Comment by bclozel** [comment]
It's actually the latest, u152 - [see release notes](http://www.oracle.com/technetwork/java/javase/8all-relnotes-2226344.html#JDK-8157561).

**Comment by bclozel** [comment]
Yes: http://hg.openjdk.java.net/jdk8u/jdk8u/jdk/file/07011844584f/src/share/lib/security/java.security-macosx
